cpp/leiden_community_detection_module/leiden_utils/leiden_utils.hpp

```diff
@@ -0,0 +1,158 @@
+#ifndef LEIDEN_UTILS_HPP
+#define LEIDEN_UTILS_HPP
```

Comment:
In the `algorithm/leiden.hpp` there is `#pragma once`, in general, under `mage` I think we use `pragma` -> align everwhere 🙏